docs/product-ibm-websphere-installation-and-configuration-guide/src/main/asciidoc/kie-server-was-group-proc.adoc

```diff
@@ -0,0 +1,20 @@
+[id='kie-server-was-goup-proc']
+= Creating the {KIE_SERVER} group and role
+
+After {KIE_SERVER} is installed, you must create the `kie-server` group and a user.
+
+.Prerequisite
+* {KIE_SERVER} is installed on the {WEBSPHERE} instance.
+//* {KIE_SERVER} nodes contain a user with `kie-server` role.
+//* Headless {PRODUCT_SHORT} controller nodes contain a user with `kie-server` role.
+
+
+.Procedure
+. In the WebSphere Integrated Solutions Console, to create the `kie-server` group, click *Users and Groups* -> *Manage Groups*.
```

Comment:
As above. I'd maybe move the 
> to create the `kie-server` group,
bit to the beginning of the step or lose it altogether but that's just me and probably not really an issue.